events/guildCreate.js

```diff
@@ -2,7 +2,7 @@
 
 module.exports = (client, guild) => {
   // Update the activity to show the updated number of guilds.
-  client.user.setActivity(`${client.settings.get("default").prefix}help | ${client.guilds.cache.size} Servers`);
```

Comment:
Not sure why this was changed.

Reply:
As we already have getSettings in function.js